app/components/CardComponent/CardComponent.js

```diff
@@ -0,0 +1,39 @@
+// React
+import React from 'react';
+// React Native
+import { Image, Text, TouchableHighlight, View} from 'react-native';
+// React Native Elements
+import { Card } from 'react-native-elements';
+// Expo
+import { LinearGradient } from 'expo-linear-gradient';
+// CSS
+import globalStyles from '../../theme/styles';
+
+class  CardComponent extends React.PureComponent {
+  constructor(props) {
```

Comment:
This component doesn't need a constructor, You can refactor it to a functional component